package.json

```
@@ -4,7 +4,9 @@
   "description": "A monorepo for the Cast app",
   "private": true,
   "scripts": {
-    "preinstall": "npx only-allow pnpm"
+    "preinstall": "npx only-allow pnpm",
+    "build:presentation": "cd presentation && pnpm build",
```

suggestion (non-blocking): `pnpm` has a `--dir` option to run commands in a particular directory. The following might be more elegant:
```suggestion
    "build:presentation": "pnpm --dir presentation build",
```

But check that it works, because I'm not sure :D